drape/gl_functions.cpp

```diff
@@ -79,7 +81,7 @@ typedef void(DP_APIENTRY * TglFlushMappedBufferRangeFn)(GLenum target, GLintptr
 
 typedef GLuint(DP_APIENTRY * TglCreateShaderFn)(GLenum type);
 typedef void(DP_APIENTRY * TglShaderSourceFn)(GLuint shaderID, GLsizei count,
-                                              GLchar const ** string, GLint const * length);
```

Comment:
In OpenGL [docs](https://registry.khronos.org/OpenGL-Refpages/gl4/html/glShaderSource.xhtml) it's said that this is the correct signature, but it's [not](https://github.com/KhronosGroup/OpenGL-Registry/blob/b53ca669bea4715b6d5fa53c459f47a1fecd7944/api/GL/glcorearb.h#L884).

Reply:
The change is already included in this PR. I'm just saying that the previous code was invalid because there's a mistake in the OpenGL documentation.